pages/about.html

```diff
@@ -100,28 +100,79 @@
         </ol>
       </nav>
     </header>
-    <main class="about-banner">
+    <main class="about-banner mx-sm-auto col-sm-10">
       <section class="text-center">
         <h1 class="h1 text-uppercase ms-lg-auto mt-5 fw-bold">
           Welcome to Hevons farms
         </h1>
         <div class="about-line"></div>
         <div class="about-txt fs-small my-5 py-4 text-dark">
-          <p
-            class="shadow-lg bg-light py-4 px-2 mx-3 col-lg-8 ms-lg-auto mx-sm-5"
-          >
+          <p class="shadow-lg bg-light py-4 px-2 mx-3 col-lg-8 ms-lg-auto">
             Hevons Farm is a subsidiary of Hevons Synergy, a company specialized
             in general contracting, Resource management, Human resources and
             Pre/Post production processing.
           </p>
           <p class="mt-5">
             Please contact us for your questions and enquiries on
             <br />
-            <a href="mailto:makojidavid@gmail.com" class="text-dark">support@hevons.com</a>
+            <a href="mailto:makojidavid@gmail.com" class="text-dark"
+              >support@hevons.com</a
+            >
           </p>
         </div>
       </section>
+      <section class="bg-white py-4">
+        <h1 class="h1 text-uppercase text-center mt-5 fw-bold">Our Values</h1>
+        <div class="about-line"></div>
+        <p class="text-center text-dark mt-3 col-sm-10 col-md-8 mx-auto">
+          We are a company built on solid cultural orientatons and believes. Our
+          goal is to ensure the availability, affordability of both raw and
+          processed/market ready food items.
+        </p>
+        <div class="shadow-lg col-6 mx-auto">
```

Comment:
- [ ] Kindly arrange the alignment of your items on your about page for the desktop version 👍 
- Your site:
![Capture](https://user-images.githubusercontent.com/85113567/181271710-a87692f6-ae80-460f-9909-0c0c8f7a5c87.PNG)
- Required design:
![Capture1](https://user-images.githubusercontent.com/85113567/181271761-1a8da2f8-352e-4e4f-bd74-0040e7eba053.PNG)

Reply:
Fixed